client/app/index.js

```diff
@@ -4,24 +4,36 @@ import VuePolyglot from './plugins/vue-polyglot'
 
 import app from './app'
 
-import foo from './components/examples/foo'
-import bar from './components/examples/bar'
+import DiscoveryList from './components/discovery_list'
+import CategoryList from './components/category_list'
+import ConnectedList from './components/connected_list'
 
 Vue.use(VueRouter)
 Vue.use(VuePolyglot)
 
 
 const routes = [
-    { path: '/foo', component: foo, alias: '/' },
-    { path: '/bar', component: bar }
-]
+  {
+    path: '/category',
+    alias: '/',
+    component: CategoryList
+  },
+  {
+    path: '/discovery',
+    component: DiscoveryList
+  },
+  {
+    path: '/connected',
+    component: ConnectedList
+  }
+];
```

Comment:
If we stuck to [StandardJS](http://standardjs.com/) (upcoming in dev environment), no semicolon needed.